### PR TITLE
added 1.17 features to enterprise overview

### PR DESCRIPTION
**Pull request by trujillo-adam**
### Description

This PR added the enterprise features releasing in 1.17 to the enterprise overview page.

### PR Checklist

* [X] external facing docs updated
* [X] appropriate backport labels added
* [X] not a security concern

### CODE REVIEW

File: website/content/docs/enterprise/index.mdx [hunk 1]
```diff
@@ -27,6 +27,8 @@ The following features are [available in several forms of Consul Enterprise](#co
 - [Automated Backups](/consul/docs/enterprise/backups): Configure the automatic backup of Consul state
 - [Redundancy Zones](/consul/docs/enterprise/redundancy): Deploy backup voting Consul servers to efficiently improve Consul fault tolerance
 - [Server request rate limits per source IP](/consul/docs/agent/limits/usage/limit-request-rates-from-ips): Limit gRPC and RPC traffic to servers for source IP addresses.
+- [Service request rate limits](/consul/docs/connect/manage-traffic/limit-request-rates): Limit the rate of HTTP requests a service receives from sources in the mesh.
```
Comment (david-yu) [inline website/content/docs/enterprise/index.mdx:30]:
```suggestion
- [Local rate limiting](/consul/docs/connect/manage-traffic/limit-request-rates): Limit the rate of HTTP requests a service receives per service instance.  
```

File: website/content/docs/enterprise/index.mdx [hunk 2]
```diff
@@ -46,6 +48,7 @@ The following features are [available in several forms of Consul Enterprise](#co
 
 - [OIDC Auth Method](/consul/docs/security/acl/auth-methods/oidc): Manage user access to Consul through an OIDC identity provider instead of Consul ACL tokens directly
 - [Audit Logging](/consul/docs/enterprise/audit-logging): Understand Consul access and usage patterns by reviewing access to the Consul HTTP API
+- Use JWTs to verify requests at the API gateway on [VMs](/consul/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-vms) and on [Kubernetes](/consul/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-k8s): Prevent unverified traffic at the API gateway using JWTs for authentication and authorization.
```
Comment (david-yu) [inline website/content/docs/enterprise/index.mdx:51]:
```suggestion
- JWT Authentication/Authorization for API Gateway: Prevent unverified traffic at the API gateway using JWTs for authentication and authorization on [VMs](/consul/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-vms) and on [Kubernetes](/consul/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-k8s):
```

File: website/content/docs/enterprise/index.mdx [hunk 3]
```diff
@@ -164,13 +172,16 @@ Consul Enterprise feature availability can change depending on your server and c
...
+| [Locality-aware routing](/consul/docs/connect/manage-traffic/route-to-local-upstreams)                   |  &#9989;  |  &#9989;   | &#10060; |
 | [Namespaces](/consul/docs/enterprise/namespaces)                        |  &#9989;  |  &#9989;   |   &#9989;  |
 | [Network Areas](/consul/docs/enterprise/federation)                     |  &#10060; |  &#10060;  |  &#10060;  |
 | [Network Segments](/consul/docs/enterprise/network-segments/network-segments-overview)            |  &#10060; |  &#10060;  |  &#10060;  |
 | [OIDC Auth Method](/consul/docs/security/acl/auth-methods/oidc)         |  &#10060; |  &#10060;  |  &#10060;  |
 | [Redundancy Zones](/consul/docs/enterprise/redundancy)                  |  n/a      |  n/a       |  n/a       |
 | [Sameness Groups](/consul/docs/connect/config-entries/sameness-group)    |  &#9989;  |  &#9989;   |   &#9989;  |
 | [Server request rate limits per source IP](/consul/docs/agent/limits/usage/limit-request-rates-from-ips) | &#9989; | &#9989; | &#9989; |
+| [Service request rate limits](/consul/docs/connect/manage-traffic/limit-request-rates) |  &#9989;  |  &#9989;   | &#10060; |
```
Comment (Jeff-Apple) [inline website/content/docs/enterprise/index.mdx:184]:
```suggestion
| [Traffic rate limiting for services](/consul/docs/connect/manage-traffic/limit-request-rates) |  &#9989;  |  &#9989;   | &#9989; |
```

**Review by Jeff-Apple — COMMENTED** [review]
Most of my suggestions are just corrections of what Enterprise features are supported on ECS.

**Review by Jeff-Apple — APPROVED** [review]

**Review by david-yu — APPROVED** [review]

**Pull request merged**